Utilities/CodeGeneration/EdFi.Ods.CodeGen/Options.cs

```diff
@@ -21,5 +22,8 @@ public class Options
 
         [Option('i', "includePlugins", Required = false, HelpText = "Flag to determine if plugin assemblies are included", Default = false)]
         public bool IncludePlugins { get; set; }
+
+        [Option('j', "extensionsPaths", Required = false, HelpText = "Array of paths for the Extension location to determine if plugin assemblies are included")]
```

Comment:
lets remove the short name for the parameter and just use `--extensionsPaths` explicitly

Reply:
yes. Fixed it